### ISSUE TITLE
[Assistive tech and device support] Alert and status messages aren't announced without receiving focus. (11.22.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28653]
#### VFS team name
Claims & Appeals

#### VFS product name
Higher Level Review

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Alert and status messages aren't announced without receiving focus.

### Issue Details
Result messages for the wizard are not being announced as they are not contained within a live region.

### Link, screenshot or steps to recreate
[Reference this previous wizard remediation ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22725)

### Supporting Guidance
[11.22.1] (https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.3_AA

---

### Platform Recommendation
Use a polite aria live region to announce results. In the future, we recommend to [use one thing per page instead of a dynamic wizard](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399) as it will be significantly more accessible.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by Mottie** [comment]
Fix has been merged and should be available in staging soon. Oddly enough, it's already available in production (see https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/)
